src/modules/rename-state.ts

```diff
@@ -0,0 +1,56 @@
+import { codeToAst } from "../parsing";
+import { selectedText, allText, selectedTextStart, selectedTextEnd, activeFileName, activeEditor, showInputBox } from "../editor";
+import traverse from "@babel/traverse";
+import { transformFromAst } from "@babel/core";
+import { window, Position, Range } from "vscode";
+import { persistFileSystemChanges, replaceTextInFile } from "../file-system";
+import { capitalizeFirstLetter } from "../utils";
+import * as t from "@babel/types";
+import { findPathInContext } from "../ast-helpers";
+
+function isPathOnLines(path, start, end) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [expected parameter: 'end' to have a typedef](https://app.codacy.com/gh/wix/vscode-glean/pullRequest?prid=5535229)